frontend/public/components/catalog/_catalog.scss

```diff
@@ -101,6 +101,11 @@ $co-modal-ignore-warning-icon-width: 30px;
     margin: 0 ($grid-gutter-width / 2) 0 $grid-gutter-width;
   }
 
+  &__button {
+    border-bottom: 1px solid $pf-color-black-300;
+    padding: 16px 24px 16px 24px;
```

Comment:
Ideally all these margin and padding values would use PF vars!

Reply:
Mary wanted custom styling for this to bring it closer to the PF4 drawer design.

Reply:
So the drawer design doesn't use PF spacing?  Why?

Reply:
Alright, it sounds like this was Mary's design solution for the buttons rather than a drawer change (there is other drawer change CSS). I'm not sure why these values were chosen.